inlong-sdk/dataproxy-sdk-twins/dataproxy-sdk-python/build.sh

```diff
@@ -45,14 +51,19 @@ cmake --build . --config Release --target check
 make check -j 4
 cd $BASE_DIR
 
-# Clone and build dataproxy-sdk-cpp
-git clone https://github.com/apache/inlong.git
-mv ./inlong/inlong-sdk/dataproxy-sdk-twins/dataproxy-sdk-cpp ./
-rm -r ./inlong
-cd ./dataproxy-sdk-cpp
-chmod +x ./build.sh
-./build.sh
-cd $BASE_DIR
+# Build dataproxy-sdk-cpp(Optional)
+read -p "Have you already built dataproxy-sdk-cpp before? (y/n): " user_choice
+
+if [ "$user_choice" != "y" ] || [ "$user_choice" != "Y" ]; then
```

Comment:
It is recommended to check whether the static library dataproxy-sdk-cpp/release/lib/libdataproxy_sdk.a has been compiled to determine whether dataproxy-sdk-cpp has been compiled.

Reply:
fixed